client/src/utils/CSSHelp.test.ts

```diff
@@ -0,0 +1,81 @@
+/* global describe it expect */
+import { cssString } from './__fixtures/curriculum-helpers-css';
+import CSSHelp from './CSSHelp';
+
+describe('CSSHelp should pass all tests..', () => {
+  document.body.innerHTML = '';
+  document.head.innerHTML = '';
+  const doc = document;
+  let t: CSSHelp;
+  beforeAll(() => {
+    const style = doc.createElement('style');
+    style.innerHTML = cssString as string;
+    doc.head.appendChild(style);
+    t = new CSSHelp(doc);
+  });
+  it('should find one `*` selector', () => {
```

Comment:
```suggestion
  it('should return a CSSStyleDeclaration array', () => {
```
The test name should describe the behaviour of the function.  It's the test callback that's responsible for asserting that it behaves that way for given fixtures, arguments etc.

Also, I'd wrap each function in a describe block.

```
describe css-help
   describe getStyleDeclaration
   describe isPropertyUsed 
```
so it's obvious what 'it' is in each case.